_posts/entities/2020-12-07-filter_momentum_collectibles.md

```diff
@@ -0,0 +1,18 @@
+---
+title: filter_momentum_collectibles
+category: entity
+tags:
+ - filter
+ - collectible system
+---
+
+----
+Filter that checks whether the player is at or above the given collectible count.
+
+
+## Keyvalues
+
+>**Minimum req. collectible count** (player_collectibles&lt;**integer**&gt;)
+
+The minimum collectible count that the player needs to be at in order for the filter to accept them.
+Default is 0.
```

Comment:
Maybe add a small note on how to add collectibles to players? That or collectibles get their own page.

Reply:
Oh yes I forgot; my idea was to close the linked issue and create another for making a guide for the collectible system as it seems to warrant that. Would that make more sense than having their own page in the entities category?

Reply:
Yes, the only new entity involved is the filter and the rest is through inputs to the player, which doesn't have its own page. So it makes more sense to create a separate page just for the collectible system; it would explain the inputs and briefly mention the filter with a link to its page.